pkg/config/config.go

```diff
@@ -58,8 +59,8 @@ func (conf *Config) SetTLS(tls TLS) {
 }
 
 // Timeout returns the HTTP request timeout in seconds once the connection is established.
-func (conf *Config) Timeout() int {
-	return conf.timeout
+func (conf *Config) Timeout() time.Duration {
+	return time.Duration(conf.timeout) * time.Second
```

Comment:
CI failed with : 
```
--- FAIL: TestFromFullConfigString (0.00s)
	assertions.go:247: 
                          
	Error Trace:	config_test.go:76
		
	Error:      	Not equal: 
		
	            	expected: int(15)
		
	            	actual  : time.Duration(15000000000)
		
	Test:       	TestFromFullConfigString
```

Additionally, moving from an int to a duration should also be done in the getter and associated field. This would make our domain object cleaner as callers won't have to think in seconds but stick to the proper Go type.

Reply:
If you want you can stick to int for this PR and I'll do a follow-up PR to change the config package.